tests/test_rules_cell.py

```diff
@@ -375,3 +375,39 @@ def test_date_lt(comparison: str, value: str, expected: bool):
 def test_greater_than_column(compared: str, data, expected: bool):
     r = GreaterThanColumn(compared)
     assert r._evaluator()(*r._prepare(data)) is expected
+
```

Comment:
The tests still need work... I didn't go back and clean them up or really work on adding any additional tests just due to (my perceived) available time to finish...